zoomto-window.cpp

```diff
@@ -0,0 +1,290 @@
+//_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/
+//_/_/
+//_/_/ AERA Visualizer
+//_/_/ 
+//_/_/ Copyright (c) 2018-2022 Jeff Thompson
+//_/_/ Copyright (c) 2018-2022 Kristinn R. Thorisson
+//_/_/ Copyright (c) 2018-2022 Icelandic Institute for Intelligent Machines
+//_/_/ Copyright (c) 2021 Karl Asgeir Geirsson
```

Comment:
This file is original by you. Maybe this was copy-pasted from another file. If you wish, you can replace Karl with
Copyright (c) 2023 .... and use your name as you want it to appear. Also please change the "general" copyright assignees years to 2018-2023.

Reply:
Good point! I just dropped that in when I first created that file, I totally forgot to update it. I just pushed a new commit that should fix this